zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/tasks/archiver/OpenSearchArchiverRepositoryIT.java

```diff
@@ -995,6 +991,42 @@ private OpenSearchAsyncClient createOpenSearchAsyncClient() {
     }
   }
 
+  private void deleteTestIndices() {
```

Comment:
❓Do we really need all these? [In the ES test we simply do a delete all.](https://github.com/camunda/camunda/blob/119dac1307183c9a8a6e1f355084907a5d179401/zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/tasks/archiver/ElasticsearchArchiverRepositoryIT.java#L96-L100) Could that work here as well?

Reply:
Makes sense, thanks!